frontends/web/src/containers/ModelStatus.js

```diff
@@ -86,6 +86,10 @@ const DeploymentStatus = ({ deploymentStatus }) => {
       description =
         "The model could not return predictions on all of the datasets. The model could have bugs.";
       break;
+    case "takendownnonactive":
+      buttonVariant = "danger";
+      description = "The model was taken down due to inactivity.";
```

Comment:
This will be changed to `"The model was taken down due to inactivity. Click `Deploy Model` to redeploy model"` in PR #805 (I'm not changing it here since until #805 is merged, there is no way to request redeployment from the UI)